pkg/api/controllers/account_controller.go

```diff
@@ -123,20 +127,35 @@ func (ctl *AccountController) GetAccount(c *gin.Context) {
 		return
 	}
 
+	if acc.Address != c.Param("address") {
```

Comment:
How it can happen?

Reply:
It can happen if the account doesn't exist, and only in this case. I changed it to:
`if acc.Address == "" {`
Is it more clear?